contrib/mesos/docs/scheduler.md

```diff
@@ -35,53 +35,59 @@ is restarted.
 A Mesos cluster can be statically partitioned using [resources roles][2]. Each
 resource is assigned such a role (`*` is the default role, if none is explicitly
 assigned in the mesos-slave command line). The Mesos master will send offers to
-frameworks for `*` resources and – optionally – for one extra role that a
-framework is assigned to. Right now only one such extra role for a framework is
+frameworks for `*` resources and – optionally – one additional role that a
+framework is assigned to. Right now only one such additional role for a framework is
 supported.
 
 ### Configuring Roles for the Scheduler
 
-Every Mesos framework scheduler can choose among the offered `*` resources and
-those of the extra role. The Kubernetes-Mesos scheduler supports this by setting
+Every Mesos framework scheduler can choose among offered `*` resources and
+optionally one additional role. The Kubernetes-Mesos scheduler supports this by setting
 the framework roles in the scheduler command line, e.g.
 
 ```bash
-$ km scheduler ... --mesos-roles="*,role1" ...
+$ km scheduler ... --mesos-framework-roles="*,role1" ...
 ```
 
-This will tell the Kubernetes-Mesos scheduler to default to using `*` resources
-if a pod is not specially assigned to another role. Moreover, the extra role
-`role1` is allowed, i.e. the Mesos master will send resources or role `role1`
-to the Kubernetes scheduler.
-
-Note the following restrictions and possibilities:
-- Due to the restrictions of Mesos, only one extra role may be provided on the
-  command line.
-- It is allowed to only pass an extra role without the `*`, e.g. `--mesos-roles=role1`.
-  This means that no `*` resources should be considered by the scheduler at all.
-- It is allowed to pass the extra role first, e.g. `--mesos-roles=role1,*`.
-  This means that `role1` is the default role for pods without special role
-  assignment (see below). But `*` resources would be considered for pods with a special `*`
-  assignment.
+This permits the Kubernetes-Mesos scheduler to accept offered resources for the `*` and `role1` roles.
+Pods may be assigned any combination of resources for those roles.
```

Comment:
By default pods may be assigned any combination of resources for those roles.
This default role assignment behavior may be overridden using the `--mesos-default-pod-roles` flag or else by applying a special annotation to the pod (demonstrated in a later section).